govuk_chat_evaluation/rag_answers/custom_deepeval/metrics/coherence/template.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import annotations
+
+DEFAULT_SCORE_RANGE = (1, 5)
```

Comment:
I don't think any of these constants are usable in a non default way so I think the naming is unnecessary